platforms/posix/src/px4/common/main.cpp

```diff
@@ -144,6 +145,7 @@ int main(int argc, char **argv)
 
 
 	if (is_client) {
+		// PX4_ERR("Starting PX4 client %s, process: %d", argv[0], getpid());
```

Comment:
Can you remove this or change it to `PX4_DEBUG`?
Below as well.

Reply:
Done.